src/css/index.less

```diff
@@ -0,0 +1,13 @@
+* {
+  letter-spacing: 0.1px;
+  -webkit-font-smoothing: antialiased;
+}
+
+i {
+  &.demisto-icon {
+    font-size: 24px !important;
+    display: inline-block;
+    margin: 0 !important;
+    vertical-align: sub !important;
+  }
+}
```

Comment:
don't do this globally,
add the relevant part only to the timer cell

Reply:
moved them to the timer cell